### ISSUE TITLE
Color Contrast Analyzer fails for default highlighted text color

### ISSUE THREAD

**Issue by ChrisSires** [dotnet/winforms#1532]
See screenshot.  Note the example shows the Windows Run dialog failing this check as well as the default winforms TextBox control.  According to this tool being used to validate MAS standards, this is a violation of MAS 1.4.3, having to do with Contrast (Minimum).  

![image](https://user-images.githubusercontent.com/37852980/62084437-2f0a4800-b20e-11e9-8013-b3c4530a34d6.png)

**Comment by ChrisSires** [comment]
See aka.ms/enable for more information on the MAS guidelines.

**Comment by merriemcgaw** [comment]
If you note, the dialog mentions that it is "just below" 4.5:1 for the contrast ratio. In the past we've been given guidance that this is most definitely just a rounding error and that if Windows is going with the same contrast then we are good to go too. We're doing an official compliance test pass right now with the accessibility group and I will confirm that they continue to approve this setting, even thought it doesn't _quite_ make 4.5:1.

**Comment by ChrisSires** [comment]
Good to hear this is a technicality and not a real issue.  Thank you.

**Comment by merriemcgaw** [comment]
Thanks!
